lib/active_shipping/carriers/fedex.rb

```diff
@@ -153,10 +161,129 @@ def find_tracking_info(tracking_number, options = {})
       parse_tracking_response(xml, options)
     end
 
+
+    # Get Shipping labels
+    # Caveats:
+    #  - Only supports singlular packages
```

Comment:
Looking at the code it seems that some carriers use `packages` and some us `package` for this method, which isn't nice. The spec in `carriers.rb` does specify an array though. I was going to suggest just using `map` to handle multiple packages but then you don't get a single response like other methods.

Not sure if it is better to use a consistent ruby API with a comment caveat or use the inconsistent ruby API with no comment caveat.